pkg/universe.dagger.io/x/contact@kjuulh.io/rust/test/image.cue

```diff
@@ -0,0 +1,39 @@
+package rust
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/x/contact@kjuulh.io/rust"
+	"universe.dagger.io/docker"
+)
+
+dagger.#Plan & {
+	actions: test: {
+		_source: dagger.#Scratch & {}
+
+		simple: {
+			_image: rust.#Image & {}
```

Comment:
Fixed. Didn't know you could just had pure rust.#Image and get the default. I guess there is a few other places where this could be cleaned up as well.